src/Bridges/Constraint/norm_to_lp.jl

```diff
@@ -0,0 +1,92 @@
+"""
+    NormInfinityBridge{T}
+
+The `NormInfinityCone` is representable with LP constraints, since ``t \\ge \\max_i |x_i|`` if and only if ``t \\ge x_i`` and ``t \\ge -x_i`` for all ``i``.
+"""
+struct NormInfinityBridge{T} <: AbstractBridge
+    ge_index::CI{MOI.VectorAffineFunction{T}, MOI.Nonnegatives}
```

Comment:
The function should be determined in `concrete_bridge_type` using `promote_operation`

Reply:
OK this means nothing to me. Can you point to an example in another cone that I can copy-paste?

Reply:
See below:
https://github.com/JuliaOpt/MathOptInterface.jl/blob/351269c98ebfeb5edeea05e0c5f393812c9d185f/src/Bridges/Constraint/scalarize.jl#L34-L38
https://github.com/JuliaOpt/MathOptInterface.jl/blob/351269c98ebfeb5edeea05e0c5f393812c9d185f/src/Bridges/Constraint/flip_sign.jl#L97-L102

Reply:
I do not understand what additional type parameters these bridges need, or the purpose of it. The geomean bridge code was my basis for implementing these new cones. Perhaps we can update these new bridges in future when the geomean bridge code is also updated?

Reply:
> I do not understand what additional type parameters these bridges need, or the purpose of it

This would allow the bridge to be used with any `AbstractVectorFunction`, e.g. `VectorQuadraticFunction`.

Reply:
So do I need `NormInfinityBridge{T, F, G}` and `NormOneBridge{T, F, G}`?

Reply:
I'm now hesitant about this because it VQF complicate duals-through-bridge: https://github.com/JuliaOpt/MathOptInterface.jl/pull/818#issuecomment-519782772